mindjunkies/forums/tests/test_views.py

```
@@ -1,197 +1,186 @@
 from mindjunkies.courses.models import Course, Module
-from mindjunkies.forums.models import ForumTopic, ForumComment, Reply
-from mindjunkies.forums.views import (
-    ForumHomeView, ForumThreadView, ForumThreadDetailsView,
-    TopicSubmissionView, CommentSubmissionView, ReplySubmissionView,
-    LikePostView, LikeCommentView, LikeReplyView
-)
-from mindjunkies.forums.forms import ForumTopicForm, ForumCommentForm, ForumReplyForm
+from mindjunkies.forums.forms import ForumCommentForm, ForumReplyForm, ForumTopicForm
+from mindjunkies.forums.models import ForumComment, ForumTopic, Reply
+from mindjunkies.forums.views import (CommentSubmissionView, ForumHomeView, ForumThreadDetailsView, ForumThreadView,
+                                      LikeCommentView, LikePostView, LikeReplyView, ReplySubmissionView,
```

Ruff (0.8.2)

`mindjunkies.forums.views.LikeCommentView` imported but unused

Remove unused import

(F401)